### DISCUSSION TITLE
Is there a way to allow loading local resources?

### DISCUSSION THREAD

**Discussion by Nongzhsh** [daijro/camoufox#35] [General]
Hello, thank you for your work on camoufox. I noticed that when initializing camoufox, some resources (camoufox browser, IP Database, etc.) need to be downloaded from Github. However, due to network restrictions in the production environment, I am unable to download these resources there. I can download them in advance, though. Is there a way to allow loading local resources?

**Comment by daijro** [comment]
You could install the library and download resources in advance through the Python CLI:

```
camoufox fetch
```

But I could add the ability to define custom paths for these resources through environment variables if needed.
